lib/tunestakeoutwrapper.rb

```diff
@@ -0,0 +1,53 @@
+# this is the one that is talking to Charles API
+require 'httparty'
+# module
+module TunesTakeout
+  BASE_URL = "https://tunes-takeout-api.herokuapp.com/"
```

Comment:
This is awesome! I like how you pulled out something you used repeatedly into a constant.